go-controller/pkg/util/ovs.go

```diff
@@ -160,12 +161,33 @@ func RunOVSVsctl(args ...string) (string, string, error) {
 	return strings.Trim(strings.TrimSpace(stdout.String()), "\""), stderr.String(), err
 }
 
+// Run the ovn-ctl command and retry if "Connection refused"
+// poll waitng for service to become available
+func runOVNretry(cmdPath string, args ...string) (*bytes.Buffer, *bytes.Buffer, error) {
+
+	for {
```

Comment:
I think we should do a exponential back off with a limit. After the limit we should quit retrying. The infinite loop feels over the top.

Reply:
I am on the fence too on this. 
@shettyg Do you have an opinion?

Reply:
Assume we have multiple masters with OVN HA. We run ovnkube as a daemonset with a replica count as 1. If for some reason, connectivity to the OVN HA cluster is not going through, I want ovnkube to crash and restart (on any new node). So we should exit after some tries.